sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -303,15 +310,21 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
   last_pc = -1
   while pc < end:
     start = pc
-    op = ord(code[pc])
-
+    try:
+      op = ord(code[pc])
+    except TypeError:
+      op = code[pc]
     if debug:
       print('-->' if pc == last_pc else '    ', end=' ')
       print(repr(pc).rjust(4), end=' ')
       print(dis.opname[op].ljust(20), end=' ')
+
     pc += 1
     if op >= dis.HAVE_ARGUMENT:
-      arg = ord(code[pc]) + ord(code[pc + 1]) * 256 + extended_arg
+      try:
```

Comment:
Maybe here (and in other places) it would make more sense to check for the version? That way its clearer that one code path is for Py2 and one is for Py3? With the try/catch its not as clear why we expect the error.

Reply:
Yeah it's more clear this way. I've changed it to use if/else.